qa-tests-backend/src/test/groovy/Services.groovy

```diff
@@ -247,20 +242,21 @@ class Services extends BaseService {
 
     static requestBuildImageScan(String registry, String remote, String tag, Boolean sendNotifications = false) {
```

Comment:
maybe this also should be moved to `ImageService`?

Reply:
It uses `detectionService` rather than `imageService`, so probably fine to leave it since there is no dedicated file for `detectionService`.